scan-command/src/main/resources/scan.properties

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# ------------------------
+# Scan Tool error messages
+# ------------------------
+error.invalid.number.of.arguments=\
+  Invalid number of arguments, expected one argument received {0}
+
+error.empty.package=\
+  Package is empty. Please add at least one .bal file
+
+error.missing.config.field=\
+  Failed to load scan tool configurations: cannot find ''configPath''
+
+error.loading.remote.config.file=\
+  Failed to load configuration file: {0}
+
+error.reading.config.file=\
+  Failed to read the configuration file: {0}
+
+error.loading.remote.platform.file=\
+  Failed to retrieve remote platform file: {0}
+
+error.downloading.remote.jar.file=\
+  Failed to download remote JAR file: {0}
+
+error.invalid.json.format=\
```

Comment:
In ballerina-lang, we have used all-simple cases for error messages. Shouldn't we follow the same behaviour here too?